src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -32,6 +33,10 @@ IOwnProps & IInjectedProps & Pick<IRootState, "auth">, IState> {
     public state: IState = {
         input: ""
     };
+
+    public async componentDidMount(): Promise<void> {
+        await BeaconChain.startAllLocalBeaconNodes();
```

Comment:
I would maybe move this to app container

Reply:
lol, makes more sense, not sure what I was thinking